### PR TITLE
Update AboutUs.md

### PR DESCRIPTION
**Pull request by janelleljt**
Update AboutUs.md with Melissa and my details.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -28,20 +28,20 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 * Responsibilities: UI
 
-### Johnny Doe
+### Melissa Harijanto
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/melissaharijanto.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/melissaharijanto)] [[portfolio](team/johndoe.md)]
```
Comment (Jiacheng-y) [inline docs/AboutUs.md]: you might want to change the portfolio link to ppp

File: docs/AboutUs.md [hunk 2]
```diff
@@ -28,20 +28,20 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-### Jean Doe
+### Janelle Loh Jen Teng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/janelleljt.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](http://github.com/janelleljt)]
 [[portfolio](team/johndoe.md)]
```
Comment (Jiacheng-y) [inline docs/AboutUs.md]: same as above

**Review by Jiacheng-y — CHANGES_REQUESTED** [review]
Apart from portfolio link, everything else looks good

**Review by Jiacheng-y — APPROVED** [review]
Looks good!

**Review by sohenze — APPROVED** [review]
Looks good

**Pull request merged**